source/use-cases/local-master-key.txt

```diff
@@ -1,23 +1,48 @@
 A. Create a Local Master Key
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Client-Side Field Level Encryption for MongoDB uses an encryption strategy called envelope encryption in which keys used to encrypt/decrypt data (called **data encryption keys**) are encrypted with another key (called the **master key**). 
+MongoDB Client-Side Field Level Encryption (CSFLE) uses an encryption strategy called envelope encryption in which keys used to encrypt/decrypt data (called **data encryption keys**) are encrypted with another key (called the **master key**). For more information on the features of envelope encryption and key management concepts, see `AWS Key Management Service Concepts <https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#enveloping`_.
 
-This strategy bolsters the security, minimizes encryption/decryption overhead, and allows the data encryption keys to be stored in the same replica set as the encrypted data. If a basic encryption strategy was used instead of envelope encryption, the computational and network load of encryption and decryption would need to be handled by the system that manages your keys. For more information on envelope encryption and key management concepts, read `this AWS article <https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#enveloping>`_.
+In CSFLE, the master key, used by the MongoDB driver to create and encrypt data keys, is securely stored in a Key Management Service (KMS). The data encryption keys, used to encrypt document fields, is stored in a user-specified collection alongside the encrypted data. 
 
-The master key is stored separately in a Key Management System (KMS) in order to prevent the data encryption keys from being used directly to access the sensitive data.
+To encrypt and decrypt document fields, you must use a client using a MongoDB driver compatible with CSFLE, and it must be able to access both the master key and data encryption keys.
```

Comment:
nit: you must use ~a client using~ a MongoDB driver ~compatible~ _that supports_ ~with~ CSFLE

Reply:
This seems like something that makes more sense under **Requirements** where we can more easily specify that this means 4.2+ drivers

Reply:
I will remove it for now. I think it could go in the *Requirements* section, but the master and data encryption keys are not explained in any of the preceding sections (intro, problem, or solution). This requirement -- client must be able to access both master key and data encryption keys -- probably fits best in *Convert to a Remote Master Key* because that makes the requirement more difficult to satisfy because without network access to the KMS, the feature will not work.